### PR TITLE
Remove Permissions From Fetcher

### PR DESCRIPTION
**Pull request by RodrigoVillar**
This PR removes permissions from `Fetcher` by having it take in a slice of strings (i.e. database keys) instead of `state.Keys`.

This PR also modifies the `Fetcher` tests to natively use database keys instead of state keys.

### CODE REVIEW

File: state/keys.go [hunk 1]
```diff
@@ -61,6 +61,15 @@ func (k Keys) ChunkSizes() ([]uint16, bool) {
 	return chunks, true
 }
 
+// Strip returns the database keys of k
+func (k Keys) Strip() []string {
```
Comment (aaronbuchwald) [inline state/keys.go:65]: Could we rename this to either `Keys()`, `KeysWithoutPermissions()`, or `WithoutPermissions()` ?

Comment (RodrigoVillar) [reply state/keys.go:65]: Will rename to `WithoutPermissions()`

Comment (RodrigoVillar) [reply state/keys.go]: Done

File: state/keys.go [hunk 2]
```diff
@@ -61,6 +61,15 @@ func (k Keys) ChunkSizes() ([]uint16, bool) {
 	return chunks, true
 }
 
+// WithoutPermissions returns the database keys of k
```
Comment (aaronbuchwald) [inline state/keys.go:64]:
```suggestion
// WithoutPermissions returns the keys of k as a slice without the permissions attached
```

**Review by aaronbuchwald — APPROVED** [review]

**Pull request merged**